presto-docs/src/main/sphinx/release/release-0.157.1-t.rst

```diff
@@ -0,0 +1,168 @@
+=================
+Release 0.157.1-t
+=================
+
+Presto 0.157.1-t is equivalent to Presto release 0.157.1, with some additional features and patches.
+
+Security
+--------
+* LDAP Authentication
```

Comment:
We need to add something like "Support for `SHOW GRANTS` added for Hive connector". Either here, or preferably, in a separate section that lists features added for hive connector.